freestyle/shared/src/test/scala/freestyle/Utils.scala

```diff
@@ -99,42 +97,42 @@ trait StateProp {
 
 object interps {
 
-  implicit val optionHandler1: FunctionK[SCtors1.Op, Option] = new SCtors1.Handler[Option] {
+  implicit val optionHandler1: FSHandler[SCtors1.Op, Option] = new SCtors1.Handler[Option] {
```

Comment:
Just wondering, do we prefer `FSHandler[SCtors1.Op, Option]` over `SCtors1.Handler[Option]` here as the type?